packages/orchestrator-ui-components/src/components/WFOWorkflowSteps/WFOStepList/WFOStepList.tsx

```diff
@@ -50,32 +62,36 @@ export const WFOStepList = React.forwardRef(
 
         return (
             <>
-                {steps.map((step, index) => {
+                {stepListItems.map((stepListItem, index) => {
                     let previousState = {};
                     let delta = {};
 
                     if (index > 0) {
                         // Todo: decided if we want this here. A refactor would be nice, or move to global state.
                         //  V1 has a rather big case statement with business logic here
-                        if (step.status === StepStatus.SUCCESS) {
-                            // Prepare a delta with the last successful step
+                        if (stepListItem.step.status === StepStatus.SUCCESS) {
+                            // Prepare a delta with the last successful stepListItem
                             let previousIndex = index - 1;
                             while (
                                 previousIndex > 0 &&
-                                (steps[previousIndex].status ===
+                                (stepListItems[previousIndex].step.status ===
                                     StepStatus.FAILED ||
-                                    steps[previousIndex].status ===
+                                    stepListItems[previousIndex].step.status ===
                                         StepStatus.WAITING)
                             ) {
                                 previousIndex--;
                             }
-                            previousState = steps[previousIndex].state;
-                            delta = stateDelta(previousState, step.state);
+                            previousState =
+                                stepListItems[previousIndex].step.state;
+                            delta = stateDelta(
+                                previousState,
+                                stepListItem.step.state,
+                            );
                         }
 
-                        // if (step.status === StepStatus.PENDING) {
-                        //     if (step.hasOwnProperty("form")) {
-                        //         Object.keys(step.form.properties as {})
+                        // if (stepListItem.step.status === StepStatus.PENDING) {
```

Comment:
Remove this commented code until we need it again

Reply:
Done